modules/ocf_tv/manifests/init.pp

```diff
@@ -18,7 +18,9 @@
       'ffmpeg',
       'feh',
       'i3',
-      'nodm',
+      'lightdm'
+      'mpc',
```

Comment:
[2020-03-17T09:15:13.098Z] Error: Could not parse for environment production: Syntax error at 'mpc' at /opt/jenkins/slave/workspace/puppet_PR-919/modules/ocf_tv/manifests/init.pp:22:7
?

Reply:
Yeah it needs a comma added at the end of the previous line

Reply:
oops lol